tests/unit/models/primo.js

```diff
@@ -2236,6 +2386,67 @@ describe("Primo Model >", function() {
 
       expect(template).toContain("Retracted Article PDF");
     });
+
+    it("should build a direct to pdf template for article search results when eoc available and expression of concern enabled", function() {
+      var scope = {
+        result: {
+          pnx: {
+            display: {
+              type: ["article"]
+            },
+
+            addata: {
+              issn: ["0020-7136"],
+              doi: ["10.1002/ijc.25451"]
+            }
+          }
+        }
+      };
+
+      var data = primo.getData(eocArticleResponse);
+      var directToPDFUrl = primo.getDirectToPDFUrl(scope, data);
+      var articleRetractionUrl = primo.getArticleRetractionUrl(scope, data);
+      var articleEOCUrl = primo.getArticleEOCNoticeUrl(scope, data);
+      var template = primo.directToPDFTemplate(directToPDFUrl, articleRetractionUrl, articleEOCUrl);
+
+      expect(data).toBeDefined();
+      expect(directToPDFUrl).toBeDefined();
+
+      expect(template).toBeDefined();
+
+      expect(template).toEqual("<div class='browzine' style='line-height: 1.4em; margin-right: 4.5em;'><a class='browzine-direct-to-pdf-link' href='https://develop.libkey.io/libraries/XXXX/10.1002/ijc.25451' target='_blank' onclick='browzine.primo.transition(event, this)'><img alt='BrowZine PDF Icon' src='https://assets.thirdiron.com/images/integrations/browzine-retraction-watch-icon.svg' class='browzine-pdf-icon' style='margin-bottom: -3px; margin-right: 1.5px;' aria-hidden='true' width='15' height='16'/> <span class='browzine-web-link-text'>Expression of Concern</span> <md-icon md-svg-icon='primo-ui:open-in-new' class='md-primoExplore-theme' aria-hidden='true' style='height: 15px; width: 15px; min-height: 15px; min-width: 15px; margin-top: -2px;'><svg width='100%' height='100%' viewBox='0 0 24 24' y='504' xmlns='http://www.w3.org/2000/svg' fit='' preserveAspectRatio='xMidYMid meet' focusable='false'><path d='M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z'></path></svg></md-icon></a></div>");
+
+      expect(template).toContain("Expression of Concern");
+      expect(template).toContain("https://develop.libkey.io/libraries/XXXX/10.1002/ijc.25451");
+      expect(template).toContain("https://assets.thirdiron.com/images/integrations/browzine-retraction-watch-icon.svg");
+    });
+
+    it("should apply the articleExpressionOfConcernText config property when eoc notice available and eoc enabled", function() {
+      browzine.articleExpressionOfConcernText = "Expression of Concern PDF";
```

Comment:
Since this link will never go directly to a PDF, but instead to LibKey Web, what if we changed the test text here to something a little different?  It can certainly be goofy 😊 

```suggestion
      browzine.articleExpressionOfConcernText = "I have a concern that must be expressed!";
```